### ISSUE TITLE
For `alert` components, informative status should be vocalized by Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1407]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

Iphone

### Device OS version

iOS 26

### OUDS iOS library version

1.3.0

### Describe the issue

For alert components, informative status need to have an accessible label, at least for Negative and Warning messages

### Expected Behavior

We should hear "Warning, Label", "Error, Label"

### Accessibility settings

VO

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.4.0**
- Build number: **1776287078**
- Build tag: **6355b9e**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
